orga/app/styles/components/progression-gauge.scss

```diff
@@ -1,12 +1,12 @@
 .progression-gauge {
-  background-color: $alto;
+  background-color: $grey-20;
   width: 100%;
   height: 4px;
   border-radius: 5px;
 }
 
 .progression-gauge__marker {
-  background: linear-gradient(to right, $amber, $orange-peel);
+  background: linear-gradient(to right, #FFBE00, #FF9F00);
```

Comment:
pas de variable couleur ou meme un gradient?

Reply:
Il ya un gradient qui a les même couleurs, mais ici c'est 'to right' exprès pour la barre de progression. En sachant que cette barre va disparaître car elle ne veut rien dire, j'ai préféré écrire cela comme ça pour le moment.